pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -16,6 +17,26 @@ const (
 	ResourceNamespaceScopedCluster = false
 )
 
+// ResourceName is the name identifying various resources in a ResourceList.
+type ResourceName string
+
+// Resource names must be not more than 63 characters, consisting of upper- or lower-case alphanumeric characters,
+// with the -, _, and . characters allowed anywhere, except the first or last character.
+// The default convention, matching that for annotations, is to use lower-case names, with dashes, rather than
+// camel case, separating compound words.
+// Fully-qualified resource typenames are constructed from a DNS-style subdomain, followed by a slash `/` and a name.
+const (
+	// ResourceCPU, in cores. (e,g. 500m = .5 cores)
+	ResourceCPU ResourceName = "cpu"
+	// ResourceMemory, in bytes. (e,g. 500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
+	ResourceMemory ResourceName = "memory"
```

Comment:
```suggestion
	// ResourceCPU in cores. (e,g. 500m = .5 cores)
	ResourceCPU ResourceName = "cpu"
	// ResourceMemory in bytes. (e,g. 500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
	ResourceMemory ResourceName = "memory"
```

Reply:
Done.

Reply:
I haven't seen it yet. Waiting for your next push.